artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/server/JMSServerManager.java

```diff
@@ -60,6 +60,24 @@ boolean createQueue(boolean storeConfig,
                        boolean durable,
                        String... bindings) throws Exception;
 
+   /**
+    * Creates a JMS Queue.
+    *
+    * @param queueName      The name of the core queue to create
+    * @param jmsQueueName the name of this JMS queue
+    * @param selectorString
+    * @param durable
+    * @return true if the queue is created or if it existed and was added to
+    * the Binding Registry
+    * @throws Exception if problems were encountered creating the queue.
+    */
+   boolean createQueue(boolean storeConfig,
```

Comment:
If this is for back compatibility for the ActiveMQDestination, why are we adding new methods to JMSServerManager, if anything "name" should be deprecated.

Reply:
The whole JMSServerManager class is already marked as deprecated.  The reason these additions are here is because the JMSServerManager was never updated to deal with the new address model since it was deprecated at the same time the address model was changed.  However, integrators which previously used the JMSServerManager may need these new methods while they transition away from using the JMSServerManager to using the ActiveMQServer instead.

Reply:
Fair enough. Thanks for the explination.